db/db_impl/db_impl_write.cc

```diff
@@ -319,11 +298,10 @@ Status DBImpl::WriteImpl(const WriteOptions& write_options,
     return status;
   }
 
-  if (immutable_db_options_.enable_multi_thread_write) {
-    std::vector<WriteBatch*> updates(1);
-    updates[0] = my_batch;
-    return MultiBatchWriteImpl(write_options, std::move(updates), callback,
-                               log_used, log_ref, seq_used);
+  if (immutable_db_options_.enable_pipelined_commit) {
+    CommitRequest request;
```

Comment:
seems it can be moved into PebbleWriteImpl

Reply:
I declare this variable here because I thought to give a twp-phase interface to order the request and then write them

Reply:
but it doesn't